store/index.js

```diff
@@ -10,9 +10,9 @@ export const state = () => ({
 });
 
 export const actions = {
-  //   async nuxtServerInit({ commit }, { $content }) {
-  //     const data = await $content('home', 'banners').fetch();
-  //     // TODO: mutate status too!
-  //     await commit('mutate', { property: 'banners', with: data });
-  //   }
+  async nuxtServerInit({ commit }, { $content }) {
```

Comment:
I think we can move this to the PageHeader component, no? I feel like it's not worth having store for this....

Reply:
just tried it out and yes! it's a `fetch` instead of an `asyncData` (since you can only do those on pages), but that seems appropriate for a banner - if there's a slight flicker there it's not terrible.